### PR TITLE
feat: Add JSON schema to `postgresql` destination plugin

### PR DESCRIPTION
**Pull request by candiduslynx**
Closes https://github.com/cloudquery/cloudquery/issues/14617

### CODE REVIEW

File: plugins/destination/postgresql/client/spec/spec.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	PgxLogLevel LogLevel `json:"pgx_log_level,omitempty" jsonschema:"default=error"`
+
+	// Maximum number of items that may be grouped together to be written in a single write.
+	BatchSize int `json:"batch_size,omitempty" jsonschema:"minimum=1,default=10000"`
+
+	// Maximum size of items that may be grouped together to be written in a single write.
+	BatchSizeBytes int `json:"batch_size_bytes,omitempty" jsonschema:"minimum=1,default=100000000"`
+
+	// Maximum interval between batch writes.
+	BatchTimeout configtype.Duration `json:"batch_timeout,omitempty" jsonschema:"default=60s"`
```
Comment (disq) [inline plugins/destination/postgresql/client/spec/spec.go:40]: Specifying `default` here is ineffective, modify it in JSONSchemaExtend

Comment (candiduslynx) [inline plugins/destination/postgresql/client/spec/spec.go:40]:
```suggestion
	BatchTimeout configtype.Duration `json:"batch_timeout,omitempty"`
```

Comment (candiduslynx) [reply plugins/destination/postgresql/client/spec/spec.go]: Done

Comment (candiduslynx) [reply plugins/destination/postgresql/client/spec/spec.go]: 9a9b6b0b433d2082e89d880c174eb556b43d99ad

**Pull request merged**